python/lib/externalrequests/dmod/externalrequests/maas_request_handlers.py

```diff
@@ -155,13 +155,23 @@ def service_url(self) -> str:
 
 class PartitionRequestHandler(MaaSRequestHandler):
 
-    def __init__(self, session_manager: SessionManager, authorizer: Authorizer, partition_service_host: str,
-                 partition_service_port: int, partition_service_ssl_dir: Path):
-        super(PartitionRequestHandler, self).__init__(session_manager=session_manager,
-                                                      authorizer=authorizer,
-                                                      service_host=partition_service_host,
-                                                      service_port=partition_service_port,
-                                                      service_ssl_dir=partition_service_ssl_dir)
+    def __init__(self, *args, **kwargs):
+        """
+
+        Parameters
+        ----------
+        args
+        kwargs
+
+        Other Parameters
+        ----------
+        session_manager
+        authorizer
+        service_host
+        service_port
+        service_ssl_dir
+        """
+        super(PartitionRequestHandler, self).__init__(*args, **kwargs)
```

Comment:
The only reason I've done this is because it's been faster ...

For some reason, this is how my JetBrains IDE auto-completes things for me.  There's probably some setting somewhere to fix this (@christophertubbs?), but it isn't terribly obvious.   It knows I'm using Python 3.8, my specified virtual environment, etc., and yet still does this.

I've started doing a little clean-up of these and need to finally take the time to sort out my IDE.

Reply:
I figured it was not done purposefully, you have a long track record of consistency and sticking to idiomatic python. That's odd that JetBrains is auto-completing it that way, weird.